Sources/Endpoints/Provisioning/Bundle ID Capabilities/EnableCapability.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  EnableCapability.swift
+//  AppStoreConnect-Swift-SDK
+//
+//  Created by Patrick Balestra on 12/23/19.
+//
+
+import Foundation
+
+extension APIEndpoint where T == BundleIdCapabilityResponse {
+
+    /// Enable a capability for a bundle ID.
+    ///
+    /// - Parameters:
+    ///   - id: (Required) An opaque resource ID that uniquely identifies the bundle identifier.
+    ///   - capabilityType: The capability type.
+    ///   - settings: An optional array of settings for this capability.
+    public static func enableCapability(
+        id: String,
+        capabilityType: CapabilityType,
+        settings: [CapabilitySetting]? = nil) -> APIEndpoint {
+
+        let request = BundleIdCapabilityCreateRequest(
+            bundleId: id,
+            capabilityType: capabilityType,
+            settings: settings
+        )
+
+        return APIEndpoint(
+            path: "bundleIdCapabilities",
+            method: .post,
+            parameters: nil,
+            body: try? JSONEncoder().encode(request))
```

Comment:
If encoding here fails, it would fail silently. This might make it harder to debug eventually if it's causing the request to fail as well. What do you think about making the `enableCapability` method throwing?

Reply:
I took inspiration from all other methods that don't propagate the failure right now. There are other 36 instances of `try? JSONEncoder().encode(request)` in the project, so not sure it's worth changing these new methods to a new convention.

Reply:
Hmm, you're right! Lets keep it like this for now.